motor/discovery/azure/resolver.go

```diff
@@ -23,9 +29,10 @@ func (r *Resolver) Name() string {
 
 func (r *Resolver) AvailableDiscoveryTargets() []string {
 	return []string{
-		common.DiscoveryAuto, common.DiscoveryAll, DiscoverySubscriptions, DiscoveryInstances,
-		DiscoverySqlServers, DiscoveryPostgresServers, DiscoveryMySqlServers, DiscoveryMariaDbServers,
-		DiscoveryStorageAccounts, DiscoveryStorageContainers, DiscoveryKeyVaults, DiscoverySecurityGroups,
+		common.DiscoveryAuto, common.DiscoveryAll, DiscoverySubscriptions,
```

Comment:
i think u could replace some of this by adding 'ResourceDiscoveryTargets...' to the list